pages/common/git-log.md

```diff
@@ -2,14 +2,18 @@
 
 > Show a history of commits.
 
-- Show a history of commits:
+- Show the sequence of commits leading to the current one:
```

Comment:
Good point. Maybe then I'd add "in reverse chronological order" to make that explicit... what do you think?